pii_redactor.sh

```diff
@@ -1,18 +1,33 @@
 #!/bin/bash
 set -eu
 # variables in use
-version="0.0.3"
+version="0.0.4"
```

Comment:
I wouldn't increment the version number until we actually release the first version.
```suggestion
version="0.0.1"
```